rust/core-lib/src/event_context.rs

```diff
@@ -344,9 +344,15 @@ impl<'a> EventContext<'a> {
         self.render()
     }
 
-    pub(crate) fn language_changed(&mut self, lang_id: &LanguageId) {
-        self.client.language_changed(self.view_id, lang_id);
-        self.plugins.iter().for_each(|plug| plug.language_changed(self.view_id, lang_id));
+    pub(crate) fn language_changed(
+        &mut self,
+        old_language_id: &LanguageId,
```

Comment:
I'm not sure we need to plumb through the old language? The plugin lib should already know what the previous language was, so if it needs that information it's available.